src/modules/explain.js

```diff
@@ -2,6 +2,7 @@ import ExplainPlanModel from 'mongodb-explain-plan-model';
 import { defaults, isString, find } from 'lodash';
 import { treeStagesChanged } from 'modules/tree-stages';
 import { globalAppRegistryEmit } from 'mongodb-redux-common/app-registry';
+import convertExplainCompat from 'mongodb-explain-compat';
```

Comment:
any chance we could have this here, these two seems to belong a lot together or?

Reply:
Acually we will monorepo that in a bit, i guess it doesn't make any difference :)

Reply:
It’s very self-contained code (as you can see), not related to any of the plugin code (or to Compass in general), so it seems to make sense to have it as a separate package and repository. I’d not include this in the monorepo either. :)

Reply:
Yeah I understand the generic purpose, i think is fine in this case.

In general I'd really love if we try to have relevant code incorporated in the project until there is a compelling practical reason not to (is reused / high value for the community (which may be the case here)), rather than doing it in the other way around. After all creating a new package is not the only way to ensure concerns are separated.